backend/routes/api.php

```diff
@@ -91,8 +91,8 @@
     Route::get('/{organization_id}', [AssessmentController::class, 'getAssByOrgId']);
     Route::put('/{assessmentId}', [AssessmentController::class, 'updateAssessment'])->middleware("isloggedin", "isadmin");
     Route::delete('/{assessmentId}/delete', [AssessmentController::class, 'deleteAssessment'])->middleware("isloggedin", "isadmin");
-    Route::get('/{companyId}/{orgId}/accepted-assessments', [AssessmentController::class, 'getCompanyAcceptedAssessments'])->middleware("isloggedin", "isadmin");
-    Route::get('/completed-assessments/{companyId}/{orgId}', [AssessmentController::class, 'getCompanyCompletedAssessments'])->middleware("isloggedin", "isadmin");
+    Route::get('/{companyId}/{userId}/accepted-assessments', [AssessmentController::class, 'getCompanyAcceptedAssessments'])->middleware("isloggedin", "isadmin");
```

Comment:
orgId should be maintained here not userId

Reply:
done